pycaching/geocaching.py

```diff
@@ -48,10 +50,31 @@ def _request(self, url, *, expect="soup", method="GET", login_check=True, **kwar
         except requests.exceptions.RequestException as e:
             raise Error("Cannot load page: {}".format(url)) from e
 
-    def login(self, username, password):
+    def login(self, username=None, password=None):
         """Logs the user in.
 
         Downloads the relevant cookies to keep the user logged in."""
+        if username is None or password is None:
```

Comment:
I think it is not important in this case (both are invalid for sure), so I prefer simpler solution.